syncd/VendorSai.cpp

```diff
@@ -26,36 +26,37 @@ VendorSai::VendorSai()
     memset(&m_apis, 0, sizeof(m_apis));
 
     sai_global_apis_t ga =
-    {
-        .api_initialize = &sai_api_initialize,
-        .api_query = &sai_api_query,
-        .api_uninitialize = &sai_api_uninitialize,
-        .bulk_get_attribute = nullptr,
+        {
+            .api_initialize = &sai_api_initialize,
+            .api_query = &sai_api_query,
+            .api_uninitialize = &sai_api_uninitialize,
+            .bulk_get_attribute = nullptr,
 #ifdef HAVE_SAI_BULK_OBJECT_CLEAR_STATS
-        .bulk_object_clear_stats = &sai_bulk_object_clear_stats,
+            .bulk_object_clear_stats = &sai_bulk_object_clear_stats,
 #else
-        .bulk_object_clear_stats = nullptr,
+            .bulk_object_clear_stats = nullptr,
 #endif
 #ifdef HAVE_SAI_BULK_OBJECT_GET_STATS
-        .bulk_object_get_stats = &sai_bulk_object_get_stats,
+            .bulk_object_get_stats = &sai_bulk_object_get_stats,
 #else
-        .bulk_object_get_stats = nullptr,
+            .bulk_object_get_stats = nullptr,
 #endif
-        .dbg_generate_dump = nullptr,
```

Comment:
why spaces are added?

Reply:
seems to be formatted by the editor. Removed.